src/app/TimeLine.tsx

```diff
@@ -12,6 +13,8 @@ export function TimeLine() {
 
   return (
     <div>
+      <Navbar/>
```

Comment:
I think Navbar component should in the Home component. We can change after current PR's when merged.

Reply:
It was written here to serve as an example